packages/utils/src/SDKErrors.ts

```diff
@@ -6,348 +6,225 @@
-  constructor() {
-    super('Owner missing')
-  }
-}
-export class ERROR_ATTESTATION_NOT_PROVIDED extends SDKError {
-  constructor() {
-    super('Attestation missing')
-  }
-}
+export class EncryptionError extends SDKError {}
 
-export class ERROR_RFA_NOT_PROVIDED extends SDKError {
-  constructor() {
-    super('RequestForAttestation missing')
-  }
-}
+export class DidError extends SDKError {}
 
-export class ERROR_LEGITIMATIONS_NOT_PROVIDED extends SDKError {
-  constructor() {
-    super('Legitimations missing')
-  }
-}
-export class ERROR_CLAIM_NONCE_MAP_NOT_PROVIDED extends SDKError {
-  constructor() {
-    super('Hashtree in Claim missing')
-  }
-}
-export class ERROR_CLAIM_NOT_PROVIDED extends SDKError {
-  constructor() {
-    super('Claim missing')
-  }
-}
-export class ERROR_ADDRESS_TYPE extends SDKError {
-  constructor() {
-    super('Address of wrong type')
-  }
-}
-export class ERROR_HASH_TYPE extends SDKError {
-  constructor() {
-    super('Hash of wrong type')
-  }
-}
+export class DidExporterError extends SDKError {}
+
+export class DidBuilderError extends SDKError {}
+
+export class Web3NameError extends SDKError {}
+
+export class ClaimHashMissingError extends SDKError {}
+
+export class RevokedTypeError extends SDKError {}
+
+export class OwnerMissingError extends SDKError {}
+
+export class AttestationMissingError extends SDKError {}
+
+export class RequestForAttestationMissingError extends SDKError {}
+
+export class LegitimationsMissingError extends SDKError {}
+
+export class ClaimNonceMapMissingError extends SDKError {}
 
-export class ERROR_HASH_MALFORMED extends SDKError {
+export class ClaimMissingError extends SDKError {}
+
+export class AddressTypeError extends SDKError {}
+
+export class HashTypeError extends SDKError {}
+
+export class HashMalformedError extends SDKError {
   constructor(hash?: string, type?: string) {
-    let message = ''
     if (hash && type) {
-      message = `Provided ${type} hash invalid or malformed \nHash: ${hash}`
+      super(`Provided ${type} hash "${hash}" is invalid or malformed`)
     } else if (hash) {
-      message = `Provided hash invalid or malformed \nHash: ${hash}`
+      super(`Provided hash "${hash}" is invalid or malformed`)
     } else {
-      message = 'Provided hash invalid or malformed'
+      super('Provided hash invalid or malformed')
     }
-    super(message)
   }
 }
 
-export class ERROR_DELEGATION_ID_TYPE extends SDKError {
-  constructor() {
-    super('DelegationId of wrong type')
-  }
-}
+export class DelegationIdTypeError extends SDKError {}
 
-export class ERROR_DELEGATION_ID_MISSING extends SDKError {
-  constructor() {
-    super('DelegationId is missing')
-  }
-}
+export class DelegationIdMissingError extends SDKError {}
 
-export class ERROR_DELEGATION_SIGNATURE_MISSING extends SDKError {
+export class DelegationSignatureMissingError extends SDKError {
   constructor() {
+    // TODO: better name
```

Comment:
Still need to address this

Reply:
In the node code we have the naming 'delegator' and 'delegate'